client/scss/components/forms/_select.scss

```diff
@@ -22,9 +22,23 @@ $chevron-inline-end-offset: math.div($select-size - $chevron-width, 2);
       theme('colors.text-label') 66%,
       transparent 66%
     );
-  background-position:
-    calc(100% - $chevron-top-offset - $chevron-width) $chevron-top-offset,
-    calc(100% - $chevron-inline-end-offset) $chevron-top-offset;
+
+  :root,
+  :host,
```

Comment:
Not sure we need host, we don't use any shadow Dom at this stage.

Reply:
I actually added :host because i saw it used in other files like https://github.com/wagtail/wagtail/blob/main/client/scss/elements/_elements.scss.

But I don't have a definite idea if it is necessary or not.

Reply:
Ok. I'll dig a bit deeper, we can leave that in for now. Thanks for explaining.